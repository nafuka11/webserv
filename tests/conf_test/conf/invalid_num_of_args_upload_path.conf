server {
    upload_path ./upload ./tmp
}