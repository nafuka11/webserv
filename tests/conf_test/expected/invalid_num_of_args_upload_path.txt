webserv: invalid number of arguments in "upload_path" directive in ./conf/invalid_num_of_args_upload_path.conf:2
